### ISSUE TITLE
DWB 1916-10_01 c8 c4 structure problems (GEBETE)

### ISSUE THREAD

**Issue by cwulfman** [pulibrary/BlueMountain#94]
from what I can tell we have a master constituent, experessed in the italics on page 42 and 48, and sub constituent prayers on the inside pages.

Move the text on page 48 to constituent 4 (it is currently part of c8) and it should be fine.

**Comment by cwulfman** [comment]
Yes, something is wrong with this constituent, the text for it is showing up in both constituents, but it only highlights for gloriosa. it should not be part of gloriosa (the mods is wrong on the pages for gloriosa) and the struct currently lists it as part of gloriosa.
The only mystery, then, is why the text shows up in the Gebete text panel at all.

**Comment by cwulfman** [comment]
The text on page 48 is still part of the wrong constituent (Gloriosa, C8) it should be part of C4 (Gebete, the parent constituent).

**Comment by cwulfman** [comment]
Moved text how you wanted.

**Comment by JNG-blue** [comment]
ok so if this has been moved it just needs to be QCed again
